conflict_resolution.html

```diff
@@ -0,0 +1,200 @@
+
+            <ul>
+                <li>Informal Mediation: This approach may include solely the parties involved in the incident, or extend to trusted individuals to help facilitate the conversation (for example, chairs or ombudspeople). The process may be more informal in that it takes place over a call or email correspondance, but the goal remains: resolution of conflict.</li>
+                <li>Formal Mediation: This approach involves the parties involved, as well as a professional mediator. It may also include parties like ombudspeople or chairs to help facilitate or provide necessary context or support. A professional mediator will be an independent third-party who is brought in to find a resolution to the conflict.</li>
+            </ul>
+
+            <section id="professional-mediation">
+                <h3>Professional Mediation</h3>
+
+                <p>In cases where it's deemed necessary to bring in a professional mediator, all parties involved:</p>
+
+                <ul>
+                    <li>will agree to attend mediation as an option for resolving the issue</li>
+                    <li>will individually attend a meeting with independent trained mediators</li>
+                    <li>will collectively attend a meeting to state how each party sees the problem and how it might be resolved</li>
+                    <li>will engage with the mediator who will work through the issues confidentially and confirm agreements with everyone as they are reached</li>
+                </ul>
+
+                <p>TODO: Details of how to access mediation</p>
+            </section>
+        </section>
+
+        <section id="investigations">
+            <h2>Investigations</h2>
+
+            <p>If resolution attempts have not worked or the issue is not appropriate for mediation, a participant can raise a formal complaint against another participant.</p>
+
+            <p>TODO: Outline where and how to send a formal complaint.</p>
+
+            <p>A formal complaint should be in writing, and include the following:</p>
+
+            <ul>
+                <li>the grounds for the complaint (for example, what parts of the CoC you think are relevant)</li>
+                <li>if appropriate, what you have done so far to try to resolve the complaint (for example, trying to solve the issue yourself or through mediation)</li>
+                <li>any evidence or applicable materials relating to the conflict (e.g., emails, minutes, screenshots, or other materials)</li>
+                <li>what resolution or action you wish to be taken to address the conflict</li>
+            </ul>
+
+            <p>An investigator will be appointed and will talk to both parties and anyone else relevant. The investigator's role is to:</p>
+
+            <ul>
+                <li>establish what happened, in a fair and objective manner</li>
```

Comment:
At some point a determination must be made about whether the CoC was violated.  Based on this draft, it sounds like this responsibility lies with the investigator.  Therefore I suggest adding another bullet here:

`<li>determine whether the incident involved a violation of the CoC</li>`